### PR TITLE
Adding inline notifications pattern

### PR DESCRIPTION
**Pull request by cshinn**
Adds the inline notifications pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
@LHinson Another one! https://www.youtube.com/watch?v=jEI3N9kIyP4

File: pattern-library/communication/inline-notifications/design/overview.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#Inline Notifications
+
+Inline notifications are used to notify a user of the status of an action during a task flow. It is recommended that inline notifications are shown at the top of the main content area.
+
+Jump to [Examples](https://www.patternfly.org/pattern-library/communication/inline-notifications/#/example-overview-1), [Variations](https://www.patternfly.org/pattern-library/communication/inline-notifications/#/example-overview-2) or [In Context](https://www.patternfly.org/pattern-library/communication/inline-notifications/#/example-overview-3)
+
+## Examples
+
```
Comment (LHinson) [inline pattern-library/communication/inline-notifications/design/overview.md]: Missing screenshots

File: pattern-library/communication/inline-notifications/design/overview.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+## Variations
```
Comment (LHinson) [inline pattern-library/communication/inline-notifications/design/overview.md]: Missing screenshots

**Comment by LHinson** [comment]
Great job @cshinn! Looks good to me. Read to merge @andresgalante?

**Comment by andresgalante** [comment]
thanks @cshinn !

**Pull request merged**